docker-compose.server.yml

```diff
@@ -6,8 +6,6 @@ services:
     hostname: redis
     networks:
       - main
-    ports:
-      - "6379:6379"
```

Comment:
Why did you get rid of these?

Reply:
I believe that @grebneerg helped with this, we were getting emails from DigitalOcean saying that our ports were exposed and they can hit them so we decided simply not to expose them.